locales/cy/stop-page.json

```diff
@@ -0,0 +1,8 @@
+{
+    "stopPageETAGPageHeader": "To be translated",
+    "stopPageETAGPageBody1": "To be translated",
+    "stopPageETAGPageBody2": "To be translated ",
+    "stopPageETAGStartServiceAgainLink": "To be translated",
+    "stopPageETAGContactUsLink": "To be translated",
```

Comment:
I don't think ETAG needs to be capitalised here

Reply:
Fixed